nrf_security/src/CMakeLists.txt

```diff
@@ -279,6 +283,16 @@ if (CONFIG_MBEDTLS_TLS_LIBRARY)
     PUBLIC
       mbedcrypto_common
   )
+
+  if (CONFIG_MBEDTLS_DEBUG)
+    # Add Zephyr's mbedlts module include path because we need rely on their headers for debug logs.
+    target_include_directories(mbedcrypto_common
+        INTERFACE
+          ${ZEPHYR_BASE}/modules/mbedtls/include
+    )
+  endif()
+  zephyr_library_sources_ifdef(CONFIG_MBEDTLS_DEBUG ${ZEPHYR_BASE}/modules/mbedtls/debug.c)
+  zephyr_library_sources_ifdef(CONFIG_MBEDTLS_DEBUG debug_init.c)
```

Comment:
Now that we have the above ifdef we can do:

zephyr_library_sources(
  debug_init.c
  ${ZEPHYR_BASE}/modules/mbedtls/debug.c
)